requirements-dev.txt

```diff
@@ -21,10 +21,10 @@ Markdown==2.6.11
 MarkupSafe==2.1.0
```

Comment:
We can remove libraries duplication by importing `requirements.txt` as we did in [Thoas](https://github.com/Ensembl/ensembl-thoas/blob/main/requirements-dev.txt).
Where we put common packages in  `requirements.txt` and the rest (related to `dev`) here in `requirements-dev.txt`

Reply:
Sounds good. Will implement that.

Reply:
@bilalebi , creating requirements-dev.txt may be out of the scope of the DC migration. I have created a ticket for this to work on after migration

Reply:
@sanjay-boddu, I agree 👍